play/dbt-materialize/README.md

```diff
@@ -0,0 +1,61 @@
+## dbt-materialize
+
+[dbt](https://www.getdbt.com/) adapter for [Materialize](http://materialize.io).
+
+Note, this plugin is a work in progress, and not yet suitable for production.
+
+### Installation
+This plugin can be installed via pip:
+```
+$ pip install dbt-materialize
+```
```

Comment:
This isn't actually true yet, right?

Reply:
Once this is merged, I think we should do a fast-follow to get this published to PyPI! Even if it's still in beta it'll be much easier to play with if it's pip-installable!

Reply:
This is misleading! I meant to write that you can `pip install` the local repo. I will fix this, but I agree with publishing to PyPI!